<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1483</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>전표정보 자동업로드 프로그램</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/favicon.png</normaloff>../img/favicon.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1461</width>
      <height>291</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <attribute name="title">
      <string>안내글</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>10</y>
        <width>1441</width>
        <height>291</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600; color:#474747;&quot;&gt;&amp;lt; &lt;/span&gt;&lt;span style=&quot; font-family:'Consolas','Courier New','monospace'; font-weight:600; color:#474747;&quot;&gt;전표등록 자동업로드 작업 프로그램 안내글 &amp;gt;&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#474747;&quot;&gt;필독 1. 반드시 안내글을 필독바랍니다.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#474747;&quot;&gt;필독 2. 해당 프로그램은 마음손 사용자 외에는 사용불가합니다.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#474747;&quot;&gt;필독 3. W4C 프로그램이 반드시 열려있는 상태여야 합니다. (회계 &amp;gt; 결의 및 전표관리 &amp;gt; 결의서/전표(간편)입력 화면을 띄워주세요.)&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#474747;&quot;&gt;1. 자동업로드 처리할 첨부파일명을 선택하세요.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#474747;&quot;&gt;2. 희망e음(또는 W4C)에 사용되는 인증코드를 입력하세요. &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#474747;&quot;&gt;3. 실행(ctrl+1) 단축키를 통해 자동화 작업을 수행합니다.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#474747;&quot;&gt;    *** '확인'을 누르면 자동화 작업이 진행되며, 작업이 완전히 종료되기 전까지 키보드 및 마우스 사용(동작)은 권장하지 않습니다.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#474747;&quot;&gt;    &lt;/span&gt;&lt;span style=&quot; color:#d43337;&quot;&gt;*** 만약, '종료'이거나 '중지' 상태가 아닌 상황에서 키보드 및 마우스 사용(동작)이 수행된 경우 나타나는 오류는 사용자가 부담합니다.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#474747;&quot;&gt;4. 중지(ctrl+9) 단축키를 통해 자동화 작업을 중지합니다.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#474747;&quot;&gt;    *** 중지된 상태에서는 더 이상 업로드 작업을 수행하지 않습니다.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#474747;&quot;&gt;    *** &lt;/span&gt;&lt;span style=&quot; color:#d43337;&quot;&gt;중지한 시점부터 자동화 업무는 실행할 수 없으며, 만약 이어서 작업이 필요한 경우 업로드 파일을 조작하여 업로드해야 하는 내역만 정리 바랍니다.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#474747;&quot;&gt;5. 엑셀파일의 사업명은 W4C에 등록된 사업명과 이름이 동일하도록 설정해주세요.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#474747;&quot;&gt;    *** 매핑해야 하는 사업명을 이미지로 저장하세요. (png 확장자만 작업 가능)&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#474747;&quot;&gt;6. 엑셀파일의 계좌명은 W4C에 등록된 계좌명과 이름이 동일하도록 설정해주세요.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#474747;&quot;&gt;    *** 매핑해야 하는 사업명을 이미지로 저장하세요. (png 확장자만 작업 가능)&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#474747;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>파일업로드</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>681</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>첨부 엑셀파일</string>
      </property>
      <widget class="QPushButton" name="find_btn">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>20</y>
         <width>61</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton:hover {
	background-color: rgb(222, 222, 222);
}

QPushButton:pressed {
	background-color: rgb(180, 180, 180);
	
}</string>
       </property>
       <property name="text">
        <string>파일찾기</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
      </widget>
      <widget class="QTextEdit" name="file_path">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>591</width>
         <height>24</height>
        </rect>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>ForbiddenCursor</cursorShape>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;D:\ysk\Python\python_auto_work\download_file\마음손거래내역_20230712053931하늘빛요양원.xlsx&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="placeholderText">
        <string>첨부파일 경로</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="file_nm">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>591</width>
         <height>24</height>
        </rect>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>ForbiddenCursor</cursorShape>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;마음손거래내역_20230712053931하늘빛요양원.xlsx&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="placeholderText">
        <string>첨부파일명</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>681</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>첨부 사업명이미지</string>
      </property>
      <widget class="QTextEdit" name="file_projectImg_path">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>591</width>
         <height>24</height>
        </rect>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>ForbiddenCursor</cursorShape>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;D:\ysk\Python\python_auto_work\download_file\일반사업.png&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="placeholderText">
        <string>첨부파일 경로</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="file_projectImg_nm">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>591</width>
         <height>24</height>
        </rect>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>ForbiddenCursor</cursorShape>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;일반사업.png&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="placeholderText">
        <string>첨부파일명</string>
       </property>
      </widget>
      <widget class="QPushButton" name="find_projectImg_btn">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>20</y>
         <width>61</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton:hover {
	background-color: rgb(222, 222, 222);
}

QPushButton:pressed {
	background-color: rgb(180, 180, 180);
	
}</string>
       </property>
       <property name="text">
        <string>파일찾기</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>681</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>첨부 계좌명이미지</string>
      </property>
      <widget class="QPushButton" name="find_manageImg_btn">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>20</y>
         <width>61</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton:hover {
	background-color: rgb(222, 222, 222);
}

QPushButton:pressed {
	background-color: rgb(180, 180, 180);
	
}</string>
       </property>
       <property name="text">
        <string>파일찾기</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
      </widget>
      <widget class="QTextEdit" name="file_manageImg_path">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>591</width>
         <height>24</height>
        </rect>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>ForbiddenCursor</cursorShape>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;D:\ysk\Python\python_auto_work\download_file\운영비통장.png&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="placeholderText">
        <string>첨부파일 경로</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="file_manageImg_nm">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>591</width>
         <height>24</height>
        </rect>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>ForbiddenCursor</cursorShape>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;운영비통장.png&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="placeholderText">
        <string>첨부파일명</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>10</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>희망e음 코드</string>
      </property>
      <widget class="QTextEdit" name="w4c_cd">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>191</width>
         <height>24</height>
        </rect>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;C0210599045&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="placeholderText">
        <string>희망e음 인증코드를 입력하세요.</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>10</y>
        <width>491</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>테스트 전용 기능(삭제예정)</string>
      </property>
      <widget class="QPushButton" name="refresh_btn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>231</width>
         <height>51</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>실행버튼을 누르면 자동화가 실행되며 더이상 마우스, 키보드 작업이 어렵습니다.</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton:hover {
	background-color: rgb(222, 222, 222);
}

QPushButton:pressed {
	background-color: rgb(180, 180, 180);
	
}</string>
       </property>
       <property name="text">
        <string>초기화(ctrl + 5)</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+5</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QTextEdit" name="refresh_path">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>471</width>
         <height>24</height>
        </rect>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;D:\ysk\Python\python_auto_work\auto_statement_upload\img\조회.png&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="start_btn">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>160</y>
        <width>161</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>HY헤드라인M</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>실행버튼을 누르면 자동화가 실행되며 더이상 마우스, 키보드 작업이 어렵습니다.</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:hover {
	background-color: rgb(222, 222, 222);
}

QPushButton:pressed {
	background-color: rgb(180, 180, 180);
	
}</string>
      </property>
      <property name="text">
       <string>실행(ctrl + 1)</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+1</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="stop_btn">
      <property name="geometry">
       <rect>
        <x>1290</x>
        <y>160</y>
        <width>161</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>HY헤드라인M</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>중지버튼을 누르면 자동화작업을 이어서 진행할 수 없습니다.</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:hover {
	background-color: rgb(222, 222, 222);
}

QPushButton:pressed {
	background-color: rgb(180, 180, 180);
	
}</string>
      </property>
      <property name="text">
       <string>중지(ctrl + 9)</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+9</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="status_text">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>930</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>종료</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>930</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>상태 :</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>1461</width>
      <height>561</height>
     </rect>
    </property>
    <property name="title">
     <string>첨부파일 내역</string>
    </property>
    <widget class="QTableWidget" name="excel_tb">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>1331</width>
       <height>491</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>0</number>
     </property>
    </widget>
    <widget class="QTableWidget" name="status_tb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>97</width>
       <height>491</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>1</number>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>56</number>
     </attribute>
     <column>
      <property name="text">
       <string>상태</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1483</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>자동업로드</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
